sci_strategic_plugin/config/parameters.yaml

```diff
@@ -1,15 +1,27 @@
 # Double: multiplier to apply to the maximum allowable vehicle deceleration limit so we plan under our capabilities
-vehicle_decel_limit_multiplier : 0.50
+vehicle_decel_limit_multiplier : 0.60
 
 # Double: multiplier to apply to the maximum allowable vehicle acceleration limit so we plan under our capabilities
 vehicle_accel_limit_multiplier : 0.50
 
 # Double: A buffer before of the stopping location which will still be considered a valid stop. Units in meters
-stop_line_buffer : 2.0
+stop_line_buffer : 10
 
 # Double: Update time interval of carma streets. Units in s
 delta_t : 1.0
 
+# Double: Update min_gap. Units in m
+min_gap : 10.0
+
+# Double: Length od the vehicle
+veh_length : 4.5
```

Comment:
This should not be a parameter here. It should be obtained either from the global /vehicle_length parameter or a TF tree lookup.

Reply:
it wasn't being used, now removed.